scripts/src/modules/timeline/century.js

```diff
@@ -0,0 +1,118 @@
+fetch(
+    "http://a08001c6c1d2e4c58bdf02589da33fca-0f850b505f873e8e.elb.eu-west-2.amazonaws.com/rosetta/data/search?aggs=decade,year,century&filter=group:community",
+) // Replace with your server-side API endpoint
```

Comment:
Not able to view the chart on the test deployed branch.
Re: was loaded over HTTPS, but requested an insecure resource. This request has been blocked; the content must be served over HTTPS.

Reply:
I'll check with Lee (K-int) on the cross domain request.